syntax = "proto3";

package sheets;

service SheetsService {
    rpc GetPitPDF (GetPitPDFRequest) returns (PitAsPDF) {}
}

message GetPitPDFRequest {
    string userId = 1;
    int32 ano = 2;
}

message PitAsPDF {
    string fileName =1;
    bytes pdf = 2;
}
